app/src/main/res/drawable/clear_night.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="32dp" android:viewportHeight="512.0"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.clear_night` appears to be unused
 Unused resources make applications larger and slow down builds.